src/main/java/org/folio/des/scheduling/quartz/QuartzExportJobScheduler.java

```diff
@@ -57,6 +57,25 @@ public List<Job> scheduleExportJob(ExportConfig exportConfig) {
     }
   }
 
+  @Override
+  public void deleteJobGroup(String tenantId) throws SchedulerException {
+    try {
+      String ediJobName = getJobGroup(tenantId);
+      Set<JobKey> jobKeySet = scheduler.getJobKeys(GroupMatcher.groupEquals(ediJobName));
+      for (JobKey jobKey : jobKeySet) {
+        scheduler.deleteJob(jobKey);
+      }
+      log.info("deleteJobGroup:: Scheduled Job Keys with size={} deleted", jobKeySet.size());
+    } catch (SchedulerException e) {
+      log.error(e.getMessage(), e);
```

Comment:
please log according to common approach. Smth like log.error("deleteJobGroup {} failed", ediJobGroup, e )